packages/client/hmi-client/package.json

```diff
@@ -56,6 +56,7 @@
 		"lodash": "4.17.21",
 		"mathlive": "0.89.2",
 		"mathml-to-latex": "1.2.0",
+		"oidc-spa": "^4.6.1",
```

Comment:
you should remove the ^ and pin to a specific version. our renovate bot will do the rest